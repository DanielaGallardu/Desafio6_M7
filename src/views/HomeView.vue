<script>
import PostsView from './PostsView.vue';
export default {
  name: 'HomeView',
  components: {
    PostsView
  }
}
</script>
<template>
  <div class="home">
    <PostsView />
  </div>
</template>
