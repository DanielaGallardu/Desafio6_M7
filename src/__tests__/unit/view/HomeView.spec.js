import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import HomeView from '@/views/HomeView.vue';

// Simulamos el componente PostsView
const PostsViewMock = {
  template: '<div class="posts-view-mock">Mock de PostsView</div>',
};

describe('HomeView.vue', () => {
  it('Contrasta Snapshot del HTML', () => {
    const wrapper = mount(HomeView, {
      global: {
        components: {
          PostsView: PostsViewMock,
        },
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
