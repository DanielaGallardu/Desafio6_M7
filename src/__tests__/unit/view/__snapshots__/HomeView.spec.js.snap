// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView.vue > Contrasta Snapshot del HTML 1`] = `
"<div class="home">
  <div class="posts-view-mock">Mock de PostsView</div>
</div>"
`;
